File: src/main/java/com/leona/guess/gameFor.java
package com.leona.guess;

import java.util.Random;
import java.util.Scanner;

public class gameFor {
    public static void main(String[] args) {
        int secret = new Random().nextInt(10) + 1;
        System.out.println(secret);
        Scanner scanner = new Scanner(System.in);
        for (int i = 1; i < 4; i++){
            System.out.print("Please input a number(" + i + "/4):");
            int number = scanner.nextInt();
            System.out.println("第" + i + "次" + number);
            if (number > secret){
                System.out.println("lower");
            }else if (number < secret){
                System.out.println("Higher");
            }else {
                System.out.println("Bingo, The number is :" + number);
                break;
            }
        }
        System.out.println("End");
    }
}

File: src/main/kotlin/com/leona/GameFor.kt
package com.leona

import java.util.*

fun main(args: Array<String>) {
    val secret = Random().nextInt(10)+1
    println(secret)
    val scanner = Scanner(System.`in`)
    for (i in 1..4){ //i from 1 to 4
        print("Please input a number: (${i}/4)")
        var number = scanner.nextInt()
        println("第${i}次$number")
        if (number > secret){
            println("Lower")
        }else if (number < secret) {
            println("Higher")
        }else{
            println("Bingo, the number is : $number")
            break
        }
    }
}